src/material-experimental/mdc-chips/chip-grid.ts

```diff
@@ -352,7 +351,11 @@ export class MatChipGrid
     }
 
     if (this._chips.length > 0) {
-      this._keyManager.setFirstCellActive();
+      // MDC sets the tabindex directly on the DOM node when the user is navigating which means
+      // that we may end up with a `0` value from a previous interaction. We reset it manually
+      // here to ensure that the state is correct.
```

Comment:
Should we file an issue for this?

Reply:
This is working as expected, as far as MDC is concerned. I don't think that it makes sense for people to customize the `tabindex` here to begin with, but we need it for backwards compatibility.